### DISCUSSION TITLE
Send music from GarageBand to Zoom and FaceTime

### DISCUSSION THREAD

**Discussion by VictorHugo2020** [ExistentialAudio/BlackHole#98] [General]
Hi, I need a little help here.
I downloaded BlackHole some days ago because I wanted to record both my voice from internal mic and audio from video chats. It worked just fine.
However, I tried to send the audio from GarageBand to Zoom and FAceTime chat. I don't know what I am doing wrong, but it is not working.
Can anybody help me to set it up and also give me a explanation about the "mechanics" behind it?
In fact, I'am not sure if I understand how to change the I/O from GarageBand, Zoom and also the Sound Preferences in System Preferences. 
I am using Macbook (Catalina).
Thank you

**Comment by devinroth** [comment]
Create Aggregate Device with BlackHole and your speakers.
Use aggregate in GB.
Route Mic in GB to BH via Aggregate.
Route GB audio to speakers and BH Aggregate.
BlackHole input to Zoom. 

Hope that makes sense.

**Comment by wilkius** [comment]
I believe I'm trying to do something similar and am struggling to make it work. 

I'm trying to combine external mic with single app audio (Qlab) and send that as the input for Discord.  I can't use a mic in Qlab so I need to route it using BlackHole via the aggregate?

Create Aggregate Device with Blackhole and Speakers - Check. (should mic be added here?)
Use Aggregate in Qlab - Check
Route Mic - Can't route in Qlab - help

Route Qlab audio to speakers and BH Aggregate - check.
Blackhole input to Discord. Check.

Feeling in over my head for something that I'm sure is not that difficult to do. Also MacBook Pro on Catalina.  Thanks in advance!!

**Comment by devinroth** [comment]
You’ll need to use something else to mix your mic into BlackHole. (I do ultimately plan on building a companion utility to help but it’s not done). With GarageBand or Logic it’s possible. Audacity is an open source solution but has known issues on Catalina. Basically you need an app to pass the output of your mic to the input of BlackHole.

**Comment by wilkius** [comment]
I was able to use LadioCast as a mixer to add the mic. I think I have that sorted now, just trying to figure out how to monitor all of this with headphones. Any suggestions?
You've done great work and it's appreciated. Thank you!!

**Comment by VictorHugo2020** [comment]
Thank you @devinroth for the answer, but I am not sure if I understood. 
I attached some figures from my settings, and maybe you can point where I need to make changes.
![Picture1](https://user-images.githubusercontent.com/63480896/79055599-a5c46580-7c24-11ea-885f-4161688ad9ad.png)

**Comment by devinroth** [comment]
How to have the audio routing setup in GB?

**Comment by VictorHugo2020** [comment]
Hi Devin. How so?

**Comment by devinroth** [comment]
Sorry I mean, could you send me a photo of what your session in GB looks like?

**Comment by VictorHugo2020** [comment]
Ok. 
<img width="1672" alt="Screen Shot 2020-04-13 at 15 51 48" src="https://user-images.githubusercontent.com/63480896/79150215-bb5d9a80-7d9e-11ea-80a4-6b971fc2a800.png">
